test/integration/cloud_event_test.ts

```diff
@@ -55,6 +55,11 @@ describe("A CloudEvent", () => {
     expect(new CloudEvent(JSON.parse(JSON.stringify(ce)))).to.deep.equal(ce);
   });
 
+  it("serializes as JSON with raw log", () => {
+    const ce = new CloudEvent({ ...fixture, data: { lunch: "tacos" } });
+    expect(ce.toString()).to.deep.equal(JSON.stringify(ce));
```

Comment:
I don't think this is actually testing the change you made. This test should pass, even without the change. E.g. this is from my local repo at `main`.

```
> ce = new CloudEvent({ source: 'test', type: 'test', data: { lunch: 'tacos' } });
CloudEvent {
  id: '7ba9e813-9bd3-4bb1-8323-4e93d8bbe4c4',
  time: '2021-09-28T21:04:04.422Z',
  type: 'test',
  source: 'test',
  specversion: '1.0',
  datacontenttype: undefined,
  subject: undefined,
  datacontentencoding: undefined,
  dataschema: undefined,
  data_base64: undefined,
  schemaurl: undefined
}
> ce.toString()
'{"id":"7ba9e813-9bd3-4bb1-8323-4e93d8bbe4c4","time":"2021-09-28T21:04:04.422Z","type":"test","source":"test","specversion":"1.0","data":{"lunch":"tacos"}}'
> JSON.stringify(ce)
'{"id":"7ba9e813-9bd3-4bb1-8323-4e93d8bbe4c4","time":"2021-09-28T21:04:04.422Z","type":"test","source":"test","specversion":"1.0","data":{"lunch":"tacos"}}'
```

I think a better test might be something like this.
```suggestion
    expect(ce.toString()).to.deep.equal(ce[Symbol.for("nodejs.util.inspect.custom")]());
```